akka-remote/src/main/scala/akka/remote/PhiAccrualFailureDetector.scala

```diff
@@ -135,7 +138,7 @@ class PhiAccrualFailureDetector(
         // don't use the first heartbeat after failure for the history, since a long pause will skew the stats
         if (isAvailable(timestamp)) {
           if (interval >= (acceptableHeartbeatPauseMillis / 2) && eventStream.isDefined)
```

Comment:
Wonder if we should increase the tolerance here somewhat. I have heard questions/complaints about this logging. Perhaps 2/3 of the acceptableHeartbeatPause?

Reply:
Yeah, half is perhaps a bit aggressive. I agree.

Reply:
#25253

Reply:
Thanks @shkoder

Reply:
good to create a separate ticket, thanks